files/en-us/mdn/contribute/howto/write_an_api_reference/index.html

```diff
@@ -253,8 +253,7 @@ <h3 id="Structure_of_an_overview_page">Structure of an overview page</h3>
 <ol>
  <li><strong>Description</strong>: the first paragraph of the landing page should provide a short, concise description of the API's overarching purpose.</li>
  <li><strong>Concepts and usage section</strong>: The next section should be titled "[name  of  API] concepts and usage", and provide an overview of all the main functionality that the API provides, what problems it solves, and how it  works — all at a high level. This section should be fairly short,  and not go into code or specific implementation details.</li>
- <li><strong>List of interfaces</strong>: This section should be titled "[name of API]  interfaces", and provide links to the reference page for each interface that makes up the API, along with a short description of what each one does. See the "Referencing other API features with the \{{domxref}} macro" section for a quicker way to create new pages. Only current interfaces should be in this list; obsolete ones should be placed in (or moved into, if obsoleted after initially being documented) the following section.</li>
- <li><strong>Obsolete interfaces:</strong> This section lists obsolete interfaces, and should be included as a subsection of the main Interfaces section.</li>
+ <li><strong>List of interfaces</strong>: This section should be titled "[name of API]  interfaces", and provide links to the reference page for each interface that makes up the API, along with a short description of what each one does. See the "Referencing other API features with the \{{domxref}} macro" section for a quicker way to create new pages. Only current interfaces should be in this list.</li>
```

Comment:
I think "only current interfaces" makes no sense when there is no talk about non-current ones anymore. I'd remove this sentence.

```suggestion
 <li><strong>List of interfaces</strong>: This section should be titled "[name of API]  interfaces", and provide links to the reference page for each interface that makes up the API, along with a short description of what each one does. See the "Referencing other API features with the \{{domxref}} macro" section for a quicker way to create new pages.</li>
```